packages/documentation/.storybook/manager.ts

```diff
@@ -1,11 +1,44 @@
 import { addons } from '@storybook/manager-api';
 import { defineCustomElement as definePostIcon } from '@swisspost/design-system-components/dist/components/post-icon.js';
 import themes from './styles/themes';
+import { API_PreparedIndexEntry, API_StatusObject } from '@storybook/types';
 
 definePostIcon();
 
-if (process.env.NODE_ENV) document.documentElement.setAttribute('data-env', process.env.NODE_ENV);
+const storedDevMode = localStorage.getItem('devModeEnabled');
 
+let initialEnv;
+if (storedDevMode !== null) {
+  initialEnv = JSON.parse(storedDevMode) ? 'development' : 'production';
+} else {
+  initialEnv = process.env.NODE_ENV || 'production';
+}
```

Comment:
While your solution is totally fine, that's how I would write it (personal preference), because it uses less code and you can read it stright forward.
It's totally ok to keep your implementation if you prefer it!

```suggestion
let initialEnv  = process.env.NODE_ENV || 'production';

if (storedDevMode !== null) {
  initialEnv = JSON.parse(storedDevMode) ? 'development' : 'production';
}
```